docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -110,8 +110,78 @@ flexLayout.WrapType = FlexLayout.FlexWrapType.NoWrap;
 layoutView.Layout = flexLayout;
 ```
 
+The following properties affect the specified child view areas:
+- `FlexAlignmentSelf`
+- `FlexPositionType`
+- `FlexAspectRatio`
+- `FlexBasis`
+- `FlexShrink`
+- `FlexGrow`
 
-## Related Information
+> [!NOTE]
+> The `FlexAlignmentSelf`, `FlexPositionType`, `FlexAspectRatio`, `FlexBasis`, `FlexShrink`, and `FlexGrow` properties are added in Tizen 6.0.
+
+**Table: Properties and descriptions**
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `FlexAlignmentSelf`    | AlignmentType   | The alignment of the item along the cross axis when free space is available. |
+| `FlexPositionType`     | PositionType    | The position type defines how the item is positioned within its parent. |
+| `FlexAspectRatio`      | float           | The ratio between the width and the height of an item. |
+| `FlexBasis`            | float           | Specifies item's default size before `FlexGrow` and `FlexShrink` calculations are performed. |
+| `FlexShrink`           | float           | Specifies how to shrink children along the main axis in case the total size of the children overflows the size of the flex layout on the main axis. |
+| `FlexGrow`             | float           | Specifies how the space within a flex layout is distributed among its children along the main axis. |
+
+## FlexAlignmentSelf property
+
+The `FlexAlignmentSelf` property has the same options and effect as the `Alignment` property, but instead of affecting the children within a flex layout, this property is applied to a single child to change its alignment within its parent. The `FlexAlignmentSelf` property overrides any option set by the parent with the `Alignment` property.
+
+The following figure shows how to use the `FlexAlignmentSelf` property:
+
+![FlexAlignmentSelf](./media/flexAlignmentSelf.png)
+
+## FlexPositionType property
+
+The `FlexPositionType` property of an element defines how it is positioned within its parent. By default, an element is positioned relatively. This means an element is positioned according to the normal flow of the layout, and its movement depends on the margin.
+The offset does not affect the position of any sibling or parent elements. When positioned absolutely, an element doesn't take part in the normal layout flow. It is instead laid out independently of its siblings.
+
+The following figure shows how to use the `FlexPositionType` property:
+
+![FlexPositionType](./media/flexPositionType.png)
+
+## FlexAspectRatio property
+
+The `FlexAspectRatio` property has the following properties:
```

Comment:
Can we say it like this. I might be wrong. Please correct me then. 
```suggestion
The `FlexAspectRatio` property has the following characteristics:
 - Accepts any floating-point value greater than `0`. The default value is undefined.
 - Defined as the ratio between the width and the height of a node, for example, if a node has an aspect ratio of `2`, then its width is twice the size of its height.
 - Has higher priority than `FlexGrow` property.
 - If aspect ratio, width, and height are set, then the cross axis dimension is overridden.

Reply:
Please see all the comments. https://github.com/Samsung/tizen-docs/pull/1392/files#r580181205
https://github.com/Samsung/tizen-docs/pull/1392/files#r580134933

Reply:
Done.